multi-node/aws/pkg/config/region_map.go

```diff
@@ -6,57 +6,22 @@ import (
 	"github.com/coreos/coreos-kubernetes/multi-node/aws/pkg/coreosutil"
 )
 
```

Comment:
Maybe we should just move this function into config.go? Not sure naming this "region_map.go" makes sense anymore.